docs/user_manual/users_and_groups.md

```diff
@@ -107,10 +89,11 @@ click the "Delete" button in the toolbar at the top of the list. For many items
 have created and modified the item are tracked. It will not be possible to delete any user who has created or edited
 any of these items. Instead, you may wish to disable the user (see above).
 
+
+
 ## Groups
 
-Groups are collections of users. Instead of assigning resource-level permissions to specific users, it may be more
-convenient to assign the permissions to a group, and then control access by adding and removing users from the group.
+Groups are collections of users.
```

Comment:
We do use groups for transfers - worth mentioning that here?

Reply:
The Transfers section talks about this and links to the Groups section. I don't think a reference/link in the other direction is as useful.